submissions/ArthurGorbenko/OOP Tiny JS World/index.js

```diff
@@ -0,0 +1,71 @@
+class inhobitant{
+  constructor(name,gender,phrase){
+    this.name = name;
+    this.gender = gender;
+    this.sayPhrase = phrase;
+  }
+  saying(){
+    print(this.sayPhrase);
+  }
+}
+
+class pet extends inhobitant {
+  constructor(name,gender,phrase,isHungry){
+    super(name,gender,phrase);
+    this.isHungry = isHungry;
+  }
+  askForFood(){
+    print("I want to eat!");
```

Comment:
`print` method should not be used in the Class method, according to the task description it should be used to list the inhabitants